docs/overrides/partials/tabs.html

```diff
@@ -13,7 +13,7 @@
       {% endfor %}
 
       <li class="md-tabs__item">
-        <a href="/api/master" class="md-tabs__link">API</a>
+        <a href="/api/10.x" class="md-tabs__link">API</a>
```

Comment:
Hm, yeah, I got the 10.x here, and in fact the build worked itself out, and the API link on drush.org is now /api/10.x after my last rebase with master.